src/v3/plugins/github/nodes.js

```diff
@@ -47,6 +47,48 @@ export type StructuredAddress =
   | CommentAddress
   | UserlikeAddress;
 
+// Each of these types has 0 or more "AUTHORS" edges,
+// each of which leads to a UserlikeAddress
+export type AuthorableAddress =
+  | IssueAddress
+  | PullAddress
+  | ReviewAddress
+  | CommentAddress;
+
+// Each of these types has text content, which means
+// it may be the source of a reference to a ReferentAddress
+export type TextContentAddress =
+  | IssueAddress
+  | PullAddress
+  | ReviewAddress
+  | CommentAddress;
+
+// Each of these types may be referred by something
+// with text content
+export type ReferentAddress =
+  | RepoAddress
+  | IssueAddress
+  | PullAddress
+  | ReviewAddress
+  | CommentAddress
+  | UserlikeAddress;
+
+// Each of these types is structurally the child of some
+// entity with a ParentAddress
+export type ChildAddress =
+  | IssueAddress // child of RepoAddress
+  | PullAddress // child of RepoAddress
+  | ReviewAddress // child of PullAddress
+  | CommentAddress; // child of IssueAddress, PullAddress, or ReviewAddress
+
+// Each of these types may have zero or more children
```

Comment:
Well, okay. I mean, this is true, but everything has zero or more
children, right? Like, a comment has zero or more children. Maybe you
mean to say, “each of these may have child nodes (see […])”?